fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1690,7 +1690,7 @@ private void buildPartitionsSequentially(long dbId, OlapTable table, List<Partit
 
     private void buildPartitionsConcurrently(long dbId, OlapTable table, List<Partition> partitions, int numReplicas,
                                              int numBackends) throws DdlException {
-        int timeout = numReplicas / numBackends * Config.tablet_create_timeout_second;
+        int timeout = numBackends * Config.tablet_create_timeout_second;
```

Comment:
why we change this logic? i think for a single tablet with multi replicas, we can have multi backends creating replicas for the tablet concurrently, that's why we devide `numBackends` here.

Reply:
I have updated it to be conditional based on the flag. If we don't special-case it then for cases when num buckets is extremely low, we end up with a timeout error message like this:

`fail to create tablet: timed out. unfinished replicas(0/0):  timeout=0s`